src/installer.js

```diff
@@ -128,7 +128,9 @@ var getDefaults = function (data, callback) {
         'GNOME',
         'GTK',
         'Utility'
-      ]
+      ],
+
+      mimeType: []
```

Comment:
This should be mimeType: ['text/plain'] by default to keep it consistent with the previous versions of the generated `.desktop` files.